libs/components/data-manager/src/lib/modules/data-manager/models/data-manager-column-picker-sort-strategy.ts

```diff
@@ -1,4 +1,14 @@
+/**
+ * These options specify the sorting applied to columns.
+ */
 export enum SkyDataManagerColumnPickerSortStrategy {
+  /**
+   * No sorting is applied to the columns.
+   */
   None = 'none',
+  /**
+   * If `columnPickerEnabled` and `sortEnabled` are set to `true`, then the columns are sorted by the order in which they are displayed, and then alphabetically. Columns that are not displayed are added after the displayed ones.
```

Comment:
It might just be me, but I don't understand what we mean by `columns are sorted by the order in which they are displayed`. What does his mean?

I don't follow the second sentence either. If columns aren't displayed, then how/where are they added after the displayed columns?

And it might just be nitpicking, but given that `sortEnabled` is mentioned in the next line but `columnPickerEnabled` isn't, it seems like we should lead with `sortEnabled` here for parallel construction.
```suggestion
   * If `sortEnabled` and `columnPickerEnabled` are set to `true`, then the columns are sorted by the order in which they are displayed, and then alphabetically. Columns that are not displayed are added after the displayed ones.
```